src/client/containers/LandingPage/LandingPage.Container.js

```diff
@@ -1,11 +1,42 @@
 import React from 'react';
 import './LandingPage.Style.css';
+import HeroImage from '../../components/HeroImage/HeroImage.component';
+import Carousel from '../../components/Carousel/Carousel.component';
+import ProductView from '../../components/ProductView/ProductView.component';
 
 const LandingPageContainer = () => {
+  const [products, setProducts] = React.useState([]);
+  const [productsImages, setProductsImages] = React.useState([]);
+  const [categories, setCategories] = React.useState([]);
+
```

Comment:
As we discussed pls add isLoading state, so you can show products when they are loaded